bin/pyactivate

```diff
@@ -54,6 +54,7 @@ def main(args: List[str]) -> int:
         python = "python3"
     else:
         python = str(activate_venv(py_dir, dev))
+        os.environ["PATH"] = str(py_dir / "venv" / "bin") + ":" + os.environ["PATH"]
```

Comment:
might want to add a comment explaining what this does

Reply:
Refactored a bit and added a comment in the new spot!